src/components/token/TokenNFTPreview.js

```diff
@@ -49,18 +49,31 @@ const TokenNFTPreview = (props) => {
   let media;
 
   if (nftType === NFT_MEDIA_TYPES.image) {
-    media = <img src={token.meta.nft.file} width="100%" height="100%" alt="NFT Preview" />;
+    media = <img src={token.meta.data.nft_media.file} width="100%" height="100%" alt="NFT Preview" />;
   } else if(nftType === NFT_MEDIA_TYPES.video && fileType) {
     media = (
-      <video controls controlsList="nodownload noremoteplayback" disablePictureInPicture onPlay={onPlayMedia}>
-        <source src={token.meta.nft.file} type={fileType} />
+      <video
+        controls
+        controlsList="nodownload noremoteplayback"
+        disablePictureInPicture
+        onPlay={onPlayMedia}
+        loop={token.meta.data.nft_media.loop}
```

Comment:
Does it have any problem if `token.meta.data.nft_media.loop` is `undefined`?

Reply:
Nope. It will works just like `false`.